Spanner/src/Operation.php

```diff
@@ -373,11 +416,17 @@ public function executeUpdateBatch(
      *         [the upstream documentation](https://cloud.google.com/spanner/docs/reference/rest/v1/RequestOptions).
      *         Please note, if using the `priority` setting you may utilize the constants available
      *         on {@see \Google\Cloud\Spanner\V1\RequestOptions\Priority} to set a value.
+     *     @type Transaction $transactionHandle Transaction to be used for this operation.
```

Comment:
I'm still seeing `transactionHandle` everywhere

Reply:
@bshaffer 

`transactionHandle` is what we decided to create instead of the reference to the `transactionId` attribute. 

We are passing the `Transaction` object inside the configuration with key `transactionHandle`


Please let me know if you need a discussion regarding this.

Reply:
See the request body here: https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances.databases.sessions/streamingRead#request-body

Reply:
Here's a PR showing how `transactionHandle` can be removed: https://github.com/googleapis/google-cloud-php/pull/6810

Reply:
Implemented alternative2.

Reply:
Implemented alternative2.